src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,180 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     * The first part.
+     *
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        Object[] arrayObj = new Object[rango[1] - rango[0] + 1];
+        arrayObj = fill(rango[0], arrayObj);
+        arrayObj = stage0(arrayObj);
+        arrayObj = stage1(arrayObj);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment; "arrayObj" already holds the assigned value along all execution paths. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4165)